package.json

```diff
@@ -37,15 +37,15 @@
     "sinon-chai": "^3.3.0",
     "tslint": "^5.18.0",
     "tslint-config-typings": "^0.3.1",
-    "typescript": "^2.0.3"
+    "typescript": "^4.5.4"
```

Comment:
This dependency update was required to get `npm run test:ts"` to pass since it is now ES2017 not ES2015